src/app/docs/tokens/tokens-docs.component.html

```diff
@@ -7,17 +7,52 @@
     The tokens component displays a series of specified objects for users to interact with. Users can navigate through the list of tokens and remove tokens from the list. The tokens component includes a message stream to place focus on tokens or remove tokens from the list.
   </sky-docs-demo-page-summary>
 
-  <!-- <sky-docs-demo>
-  </sky-docs-demo>
+  <sky-docs-demo>
+    <sky-docs-demo-control-panel
+      (selectionChange)="onDemoSelectionChange($event)"
+    >
+      <sky-docs-demo-control-panel-section>
+        <sky-docs-demo-control-panel-checkbox
+          label="Disabled"
+          propertyName="disabled"
+          [checked]="false"
+        >
+        </sky-docs-demo-control-panel-checkbox>
+        <sky-docs-demo-control-panel-checkbox
+          label="Dismissible"
+          propertyName="dismissible"
+          [checked]="true"
+        >
+        </sky-docs-demo-control-panel-checkbox>
+        <sky-docs-demo-control-panel-checkbox
+          label="Focusable"
```

Comment:
Is it correct behavior that the when both the focusable and dismissible options are selected and you tab through the tokens, the focus alternates between the tokens and the close button?

Also, should the *Reset* option reload tokens that have been closed?

Reply:
Yes, that's by design.

Good idea about the reset button. I'll implement that.